templates/skeleton/app/entry.server.tsx

```diff
@@ -1,6 +1,6 @@
 import type {EntryContext, AppLoadContext} from '@shopify/remix-oxygen';
 import {RemixServer} from '@remix-run/react';
-import isbot from 'isbot';
+import isbotua from 'isbot';
```

Comment:
Would prefer to update the `isbot` package instead of fixing for lint error. The later version of `isbot` will not have the default export. https://www.npmjs.com/package/isbot

```diff
- import isbot from 'isbot';
+ import {isbot} from 'isbot';
```